package.json

```diff
@@ -19,9 +19,12 @@
   },
   "dependencies": {
     "expo": "^32.0.0",
+    "node": "^12.2.0",
```

Comment:
this is creating a conflict 
<img width="518" alt="Screenshot 2019-05-17 at 16 32 51" src="https://user-images.githubusercontent.com/43641321/57935218-9aa47080-78c1-11e9-8df2-4692ad52d50c.png">

Reply:
Ok. Please remove the node package from the projects package.json. Node should be used from your global system packages.

Reply:
Ok. Please remove the node package from the projects package.json. Node should be used from your global system packages.

Reply:
Resolved with cherry picking the [commit](https://github.com/ikuseiGmbH/smart-village-app-app/commit/cc2880c3fe49f4f1a374df8c3aaebd144d65b4fa) from branch: **feauture/working-on-components**